bot/exts/recruitment/talentpool/_review.py

```diff
@@ -433,11 +434,20 @@ async def _previous_nominations_review(self, member: Member) -> Optional[str]:
 
         nomination_times = f"{num_entries} times" if num_entries > 1 else "once"
         rejection_times = f"{len(history)} times" if len(history) > 1 else "once"
+        threads = [await get_or_fetch_channel(nomination.thread_id) for nomination in history]
+
+        nomination_vote_threads = ", ".join(
+            [
+                f"[Thread-{thread_number}]({thread.jump_url})" if thread else ''
+                for thread_number, thread in enumerate(threads, start=1)
+            ]
+        )
```

Comment:
We should also catch the error if the channel isn't found here. Also, we can't use the `[]()` syntax for links as this isn't sent in an embed.

Reply:
Still this 
> Also, we can't use the `[]()` syntax for links as this isn't sent in an embed.